board/recalbox/fsoverlay/recalbox/share_init/system/.emulationstation/es_systems.cfg

```diff
@@ -67,6 +67,7 @@
 		    <core>gliden64</core>
 		    <core>n64</core>
 		    <core>rice</core>
+    		    <core>glide64mk2</core>
```

Comment:
duplicate the file or create an xml to show supported system.
while it's in advanced feature, i decided to let advanced people known what
they do ;-)

2016-06-04 9:13 GMT+02:00 digitalLumberjack <notifications@github.com>:

> In
> board/recalbox/fsoverlay/recalbox/share_init/system/.emulationstation/es_systems.cfg
> <https://github.com/recalbox/recalbox-buildroot/pull/367#discussion_r65798764>
> :
>
> > @@ -67,6 +67,7 @@
> >  		    <core>gliden64</core>
> >  		    <core>n64</core>
> >  		    <core>rice</core>
> > +    		    <core>glide64mk2</core>
>
> We let that available for every system. Any idea to avoid that on rpi ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/recalbox/recalbox-buildroot/pull/367/files/84cbb069e17dde9c1972a2a7f7d3d267c25b0b53#r65798764>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFGWYcKIMfEb1Vpude4WssKDaeZXVCBPks5qISWYgaJpZM4Is1I6>
> .
>